### ISSUE TITLE
[Toggle button] Toggle-button should use `aria-labelledby` to connect the tooltip

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#3005]
### Provide a general summary of the issue here

Current version of the `toggle button` uses `aria-label` and based on provided `aria-label` we set a tooltip on the button. But the tooltip should be set by `aria-labelledby` (since those are icon-only buttons), right now it's set by `aria-describedby`:

<img width="846" height="546" alt="Image" src="https://github.com/user-attachments/assets/3904bbde-c325-461a-a4e9-6cdfc09bebf0" />

\
So probably `ariaRelation: 'label'` needs to be set on the `Tooltip.lazy` (to have the tooltip connected by `aria-labelledby`). 
But in that case `aria-label` and `aria-labelledby` would be present on the `toggle button`.

Please make sure that there is **only `aria-labelledby`** set on the `toggle button` (so no `aria-label`). 

Since this component is in **_preview_** status, please don't change the current API 
(so keep `@property({ reflect: true, attribute: 'aria-label' }) label?: string;` as it is).

### 👤 Your name

SLDS Team

### 🧢 Your Product/Team

SLDS
